### PR TITLE
dev-python/PythonQt: prevent using with Qt greater then 5.7, bump to 3.2

### PR DESCRIPTION
**Pull request by SpiderX**
1. Added Python 3.6 to PYTHON_COMPAT.
2. Restricted subslot for Qt packages to 5.7 (wrt #634544)

Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

File: dev-python/PythonQt/PythonQt-3.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+HOMEPAGE="http://pythonqt.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythonqt/pythonqt/${P}/${MY_P}.zip"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc +extensions webkit"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-qt/qtcore:5/5.7
```
Comment (SoapGentoo) [inline dev-python/PythonQt/PythonQt-3.1-r1.ebuild]: please use version specifier (`<dev-qt/qtcore-5.7:5`) instead of pinning to subslots

Comment (SpiderX) [reply dev-python/PythonQt/PythonQt-3.1-r1.ebuild]: Done.

**Comment by SoapGentoo** [comment]
@SpiderX looks good, one thing I forgot to mention, please to the version fixing and version bumping in two separate commits, for easier reverting ;)

**Comment by SpiderX** [comment]
Divided the commit into two.

**Pull request closed without merging**